src/doc/en/developer/coding_basics.rst

```diff
@@ -254,14 +254,14 @@ The top of each Sage code file should follow this format::
 
     AUTHORS:
 
-    - YOUR NAME (2005-01-03): initial version
-
-    - person (date in ISO year-month-day format): short desc
+    - Your Name (2024-01-13): initial version
+    - Alice Liddell (2024-05-31): added a method; cleaned docstrings
+    - Full Name (YYYY-MM-DD): short description
 
```

Comment:
```suggestion
```

Reply:
Oh! one more thing please...

Reply:
OK.  

Actually I inserted a blank line after the list intentionally to surround the list by blank lines, though it is strictly not necessary.  This is another point where people (or one person at different times :-) use different conventions.

However, I feel that no blank line at the end of the text is the trend, and we can save a line (perhaps useless).